nucypher/utilities/networking.py

```diff
@@ -145,10 +158,10 @@ def get_external_ip_from_known_nodes(known_nodes: FleetSensor,
     if len(known_nodes) < sample_size:
         return  # There are too few known nodes
     sample = random.sample(list(known_nodes), sample_size)
+    client = NucypherMiddlewareClient()
     for node in sample:
-        ip = _request(url=node.rest_url())
+        ip = _request_from_node(teacher=node, client=client)
         if ip:
-            log.info(f'Fetched external IP address ({ip}) from randomly selected known node(s).')
```

Comment:
I think log messages were potentially useful. Any particular reason for removing them?

Reply:
Hmm, unintentionally removed in this case.  I agree they are useful and want to restore it.

Reply:
:heavy_check_mark: